internal/api/services.go

```diff
@@ -127,3 +140,8 @@ func (client *Client) DeleteService(ctx context.Context, id string) (*any, error
 	url := fmt.Sprintf("%s/services/%s", client.BaseURLV3, id)
 	return Request[any, any](http.MethodDelete, url, client, ctx, nil)
 }
+
+func (client *Client) UpdateSlackChannel(ctx context.Context, serviceID string, req *AddSlackChannelReq) (*any, error) {
```

Comment:
Does creating service support slack channel setup?
Making 2 api calls per resource is considered an anti-pattern in tf.

Reply:
No, we don't support that

Reply:
ok